gen/nvml/device.go

```diff
@@ -933,18 +933,44 @@ func (Device Device) GetBridgeChipInfo() (BridgeChipHierarchy, Return) {
 
 // nvml.DeviceGetComputeRunningProcesses()
 func DeviceGetComputeRunningProcesses(Device Device) ([]ProcessInfo, Return) {
+	var Infos []ProcessInfo  // This is the v2 version of process info data structure
 	var InfoCount uint32 = 1 // Will be reduced upon returning
+	var ret = SUCCESS        // Will be changed upon returning
 	for {
-		Infos := make([]ProcessInfo, InfoCount)
-		ret := nvmlDeviceGetComputeRunningProcesses(Device, &InfoCount, &Infos[0])
+		if usesNvmlDeviceGetComputeRunningProcesses_v1 {
+			var v1Infos = make([]ProcessInfo_v1, InfoCount)
+			ret = nvmlDeviceGetComputeRunningProcesses_v1(Device, &InfoCount, (*ProcessInfo)(unsafe.Pointer(&v1Infos[0]))) // Call v1 version
+			if ret == SUCCESS && InfoCount > 0 {
+				// Convert process info data structure from v1 to v2
+				for i := range v1Infos {
```

Comment:
I think we would need to use `InfoCount` as the loop variable here as this is modified in the call above to return the actual number of elements.

Reply:
Thanks. I'll fix this right away.